app/json/slate_lineup_configs.json

```diff
@@ -513,5 +513,27 @@
         ]
       }
     ]
+  },
+  {
+    "id": "a26db39b-a68a-4e01-b680-443a466f9c36",
```

Comment:
i just generated this in the python console - is that okay?

Reply:
Yes. At some point we discussed these IDs being a hash of the object, but then we truncated it, at which point it doesn't represent the object anymore. Random seems fine.

Reply:
Just to share knowledge: the slate/lineup ids are static GUIDs, the only requirement is that they are unique w.r.t the namespace. The experiment_ids are truncated hashes based on the slate/lineup configuration, and are auto-generated.